api/src/main/java/com/cloud/agent/api/to/S3TO.java

```diff
@@ -201,7 +201,7 @@ public DataStoreRole getRole() {
         return DataStoreRole.Image;
     }
 
-    public boolean getEnableRRS() {
+    public boolean isEnableRRS() {
```

Comment:
what is the is prefix for in this case? is the question enableRRS() ?

Reply:
Does not `enableRSS` give the idea that it is a method that "enables" RSS?
At least, that is what sounds to me.